docs/fsharp/language-reference/xml-documentation.md

```diff
@@ -3,54 +3,149 @@ title: XML Documentation
 description: Learn about support in F# for generating documentation from comments.
 ms.date: 05/16/2016
 ---
-# XML Documentation
+
+# Document your code with XML comments
 
 You can produce documentation from triple-slash (///) code comments in F#. XML comments can precede declarations in code files (.fs) or signature (.fsi) files.
 
-## Generating Documentation from Comments
+XML documentation comments are a special kind of comment, added above the definition of any user-defined type or member.
+They are special because they can be processed by the compiler to generate an XML documentation file at compile time.
+The compiler-generated XML file can be distributed alongside your .NET assembly so that IDEs can use tooltips to
+show quick information about types or members. Additionally, the XML file can be run through tools
+like [fsdocs](http://fsprojects.github.io/FSharp.Formatting/) to generate API reference websites.
+
+XML documentation comments, like all other comments, are ignored by the compiler, unless the options described below are enabled to check the validity and
+completeness of comments at compile-time.
+
+You can generate the XML file at compile time by doing one of the following:
+
+- If you are developing an application with .NET Core from the command line, you can
+  add a `GenerateDocumentationFile` element to the `<PropertyGroup>` section of your .fsproj project file. You can
+  also specify the path to the documentation file directly using [`DocumentationFile` element](/visualstudio/msbuild/common-msbuild-project-properties). The
+  following example generates an XML file in the project directory with the same root filename as the assembly:
+
+   ```xml
+   <GenerateDocumentationFile>true</GenerateDocumentationFile>
+   ```
+
+   This is equivalent to the following:
+
+   ```xml
+   <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\$(AssemblyName).xml</DocumentationFile>
```

Comment:
This should just be removed. We shouldn't encourage anyone using the older property as it was only built to be an implementation detail for Visual Studio to work out.